lib/dependency_container.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:dio/dio.dart';
+import 'package:get_it/get_it.dart';
+import 'package:marvelapp_flutter/data/data_sources/local/dao/character_dao.dart';
+import 'package:marvelapp_flutter/data/data_sources/local/database/database.dart';
+import 'package:marvelapp_flutter/data/data_sources/local/character_data_source.dart';
+import 'package:marvelapp_flutter/data/data_sources/remote/remote_data_source.dart';
+import 'package:marvelapp_flutter/data/repository/dio_marvell_repository.dart';
+
+final getIt = GetIt.instance;
+
+void setup() {
+  getIt.registerLazySingleton<Database>(() => Database());
```

Comment:
Why you do not added the use cases here?

Reply:
done